site/docs/v1/tech/guides/configuring-application-specific-email-templates.adoc

```diff
@@ -0,0 +1,91 @@
+
+Make sure you've created an Application in the link:/docs/v1/tech/core-concepts/applications[Application overview] guide. After, log in to your application by clicking on `Applications` in the navigation sidebar, select the green magnifying glass icon, and copy and paste the `Login URL` into an incognito browser. Click the Forgot Password link, enter your email in the form, and you should see the `Forgot Password` email sent to your inbox.
+
+### Overriding email templates
+
+Finally, we'll override our generic Tenant Email Template we just configured with a custom application specific template for a fictitious company called Pied Piper.
+
+Navigate to [breadcrumb]#Customizations -> Emails Templates# and click the green + icon to create a new template.
+Fill out the `Name`, `Default Subject`, `From Email`, `Default from Name` fields.
+Paste the below code snippet into the `HTML Template` section and modify based on your application:
+
+```
+[#setting url_escaping_charset="UTF-8"]
+To change your password for your Pied Piper account, click on the following link.
+<p>
+[#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state'.
+If we have an application context, append the client_id to ensure the correct application theme when applicable.
+--]
+[#assign url = "http://localhost:9011/password/change/${changePasswordId}?client_id=${(application.oauthConfiguration.clientId)!''}&tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && key != "client_id" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+<a href="${url}">${url}</a>
+</p>
+- Pied Piper Admin
+
+```
+
+and for `Text Template` section, we'll use:
+
+```
+[#setting url_escaping_charset="UTF-8"]
+To change your password for your Pied Piper account, click on the following link.
+
+[#-- The optional 'state' map provided on the Forgot Password API call is exposed in the template as 'state' --]
+[#assign url = "https://local.fusionauth.io/password/change/${changePasswordId}?tenantId=${user.tenantId}" /]
+[#list state!{} as key, value][#if key != "tenantId" && value??][#assign url = url + "&" + key?url + "=" + value?url/][/#if][/#list]
+
+${url}
+
+- Pied Piper Admin
+
+```
+
+Save your template by clicking the blue save icon in the top right corner.
+
+### Test our custom template
+
+For our demo application Pied Piper, we'll call the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[Forgot Password API] for an existing user. If you haven't already created a user, go ahead and complete the process in the  link:/docs/v1/tech/tutorials/register-user-login-api[Register a User and Login] guide.
```

Comment:
Why wouldn't you go back to the UX the way you did before?

Reply:
`applicationId` is documented through our APIs but its not that apparent how to override the Tenant email templates. I can also include the option of using the hosted Login URL.

Reply:
Added a section for appending the `applicationId` to the Login URL. @mooreds